webpack-test/webpack.config.dev.js

```diff
@@ -0,0 +1,38 @@
+const path = require("path");
+const HtmlWebpackPlagin = require("html-webpack-plugin");
+
+module.exports = {
```

Comment:
Not an error, but you can try to use [webpack-merge](https://www.npmjs.com/package/webpack-merge). It will reduce repeating code in configuration

Reply:
Fixed